src/lib/Preview.js

```diff
@@ -265,6 +265,9 @@ class Preview extends EventEmitter {
         // Clean the UI
         this.ui.cleanup();
 
+        // Eject http interceptors
+        api.ejectInterceptors();
```

Comment:
If all Preview instances are sharing the same interceptors, destroying them when one is hidden seems counter-intuitive. Is that the behavior users would expect?

Reply:
Seems like the right move is to move instantiation further up the chain like the constructor and tear down in the de-constructor. That will require a small change to UI elements